R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
also please note my comment earlier-  isNaN exists before 1.6 - if we remove it, we should explain this in release note.